### PR TITLE
Employee details view page updation 1

### PR DESCRIPTION
**Pull request by VINCENT08122001**

### CODE REVIEW

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+@Component({
+  selector: 'app-profile-details',
+  templateUrl: './profile-details.component.html',
+  styleUrls: ['./profile-details.component.css'],
+})
+export class ProfileDetailsComponent {
+  first: number = 0;
+  rows: number = 10;
+  onPageChange(event: any) {
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: write the function below

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  Inactive: any = {};
+  newdata: Array<ProfileInterface> = [];
+  visible: boolean = false;
+  ingredient!: boolean;
+  obj: any = {};
+  constructor(private datum: ProfileDetailServiceService) {}
+
+  ngOnInit(): void {
+    this.datum.getData().subscribe((data: ProfileInterface[]) => {
+      console.log(data);
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: remove these logs

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  rows: number = 10;
+  onPageChange(event: any) {
+    this.first = event.first;
+    this.rows = event.rows;
+  }
+
+  array: Array<ProfileInterface> = [];
+  Inactive: any = {};
+  newdata: Array<ProfileInterface> = [];
+  visible: boolean = false;
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: declared but never used

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  ingredient!: boolean;
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: declared but never used

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  obj: any = {};
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: declared but never used

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  first: number = 0;
+  rows: number = 10;
+  onPageChange(event: any) {
+    this.first = event.first;
+    this.rows = event.rows;
+  }
+
+  array: Array<ProfileInterface> = [];
+  Inactive: any = {};
+  newdata: Array<ProfileInterface> = [];
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: declared but never used

File: src/app/employee-details/profile-details/profile-details.component.ts [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  ngOnInit(): void {
+    this.datum.getData().subscribe((data: ProfileInterface[]) => {
+      console.log(data);
+      this.array = data;
+      console.log(this.array);
+    });
+  }
+
+  showDialog(array: any) {
+    this.visible = true;
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.ts]: there is't any need for this line

File: src/app/employee-details/profile-details/profile-details.component.html [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+        <td>{{ array.qualification.toUpperCase() }}</td>
+        <td>{{ array.CTC }}</td>
+        <td>{{ array.email }}</td>
+        <td>{{ array.experience.toUpperCase() }}</td>
+        <td>{{ array.totalyear }}</td>
+        <td>{{ array.emergencyContactNumber }}</td>
+
+        <td>
+          <p-button
+            (click)="showDialog(array)"
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.html]: give some meaningfull name

File: src/app/employee-details/profile-details/profile-details.component.css [hunk 9]
```diff
@@ -0,0 +1,21 @@
+#headding {
```
Comment (HarishDass) [inline src/app/employee-details/profile-details/profile-details.component.css]: meaningful name

**Review by aarthi1246 — APPROVED** [review]

**Pull request closed without merging**